compiler/scenario-service/protos/scenario_service.proto

```diff
@@ -240,8 +240,9 @@ message ScenarioError {
     // Error was raised via the error builtin
     string user_error = 13;
 
-    // Unhandled exception
-    Value unhandled_exception = 26;
+    // Unhandled exceptions
+    string unhandled_builtin_exception = 26;
+    Value unhandled_user_exception = 27;
```

Comment:
Have you considered having an `Exception` type instead and then reference that from the unhandled exception error?

Reply:
Do you prefer like I have done in beb987acf0bc13257566576cf6e446de96ef8cee ?